editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Node.tsx

```diff
@@ -93,6 +93,10 @@ const Node: React.FC<any> = (props) => {
       );
     case TYPES.Review:
       return <Question {...allProps} text={node?.data?.title ?? "Review"} />;
+    case TYPES.Section:
+      return (
+        <Question {...allProps} text={node?.data?.title ?? "Section marker"} />
+      );
```

Comment:
No special styling here yet, but this is where we'll want to edit if we want sections to have a unique Editor style representation. See `hasFailed` and `wasVisited` props on this existing `Question` component as example approaches to styling that we could follow, or we could roll an alternative to this `Question` altogether.

I quickly prototyped this style with an emphasized bottom border, but wasn't sold & figured this is better revisited later.
![Screenshot from 2023-02-21 08-45-28](https://user-images.githubusercontent.com/5132349/220280462-7528471a-aeda-4d76-a5c9-9672785b0565.png)

Reply:
I think this is great for now - one for Alastair an Ian I think!